src/languages/index.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const path = require('path');
+
+const translationsPath = path.join(__dirname, 'translations');
+
+function loadTranslations(lang) {
+  try {
+    const filePath = path.join(translationsPath, `${lang}.json`);
+    if (!fs.existsSync(filePath)) {
+      throw new Error(`Translation file for language '${lang}' not found.`);
+    }
+    return JSON.parse(fs.readFileSync(filePath, 'utf-8'));
+  } catch (error) {
+    console.error(error.message);
+    return null;
+  }
+}
+
+function translateLanguage(lang, key) {
+  const translations = loadTranslations(lang);
+  if (!translations) {
```

Comment:
just in case translations was an empty object
```suggestion
  if (!translations || !Object.values(translations).length) {
```